integration-cli/docker_cli_stat_test.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"os/exec"
+	"testing"
+	"time"
+)
+
+func TestCliStatsNoStream(t *testing.T) {
+	defer deleteAllContainers()
+	var (
+		name   = "statscontainer"
+		runCmd = exec.Command(dockerBinary, "run", "-d", "--name", name, "busybox", "top")
+	)
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatalf("Error on container creation: %q, output: %v", err, out)
```

Comment:
should be `%v` and `%s` in that order. %q makes it harder to read with all the backslashes.